include/deal.II/fe/mapping.h

```diff
@@ -475,7 +474,20 @@ class Mapping : public Subscriptor
     virtual ~InternalDataBase ();
 
     /**
-     * Values updated on each cell by Mapping::fill_fe_values() and friends.
+     * A set of update flags specifying the kind of information that
+     * an implementation of the Mapping interface needs to compute on
+     * each cell or face, i.e., in Mapping::fill_fe_values() and
+     * friends.
+     *
+     * This set of flags is stored in implementations of
+     * Mapping::get_data(), Mapping::get_face_data(), or
+     * Mapping::get_subface_data(), and is that subsets of the update
+     * flags passed to those functions that require re-computation on
+     * every cell. (The subset of the flags corresponding to
+     * information that can be computed once and for all already at
+     * the time of the call to Mapping::get_data() -- or an
+     * implementation of that interface -- need not be stored here
+     * because it has already been taken care of.)
```

Comment:
I think this is already addressed by #1361, so I guess this gets removed once #1361 is merged.